### ISSUE TITLE
Pygount is printing blank lines

### ISSUE THREAD

**Issue by susumuasaga** [roskakori/pygount#91]
Windows 11
Powershell
Version 1.4.0

After I upgraded, Pygount is printing blank lines:
```
C:\> pygount src









```

I downgrade to version 1.3.0 and it is working now:
```
C:\> pygount src 
0       __unknown__     src     src\spark_tests.egg-info\PKG-INFO
28      Text only       src     src\spark_tests.egg-info\SOURCES.txt
0       Text only       src     src\spark_tests.egg-info\dependency_links.txt
1       Text only       src     src\spark_tests.egg-info\top_level.txt
0       __empty__       src     src\spark_tests\__init__.py
16      Python  src     src\spark_tests\datetime.py
72      Python  src     src\spark_tests\delta.py
0       __unknown__     src     src\spark_tests\sql.puml
113     Python  src     src\spark_tests\sql.py
```

**Comment by roskakori** [comment]
Hm, might be related to the recent switch to `rich` to show the output as table. 🤔 

Sadly I'm unable to investigate this with a setup similar to yours as I don't access to any Windows 11 box.

You could try to upgrade `rich` to the most current version:
```bash
pip install --upgrade rich
```

As a workaround, keep using version 1.3.0 until someone can look into this.

**Comment by fkubicek** [comment]
I'm on Windows 10 and I also get just a bunch of blank lines instead of any output. Downgrading to 1.3.0 fixes it.

**Comment by bkelly312** [comment]
Confirmed on Windows 10 Powershell, pygount 1.4.0
Upgrading rich did not help.

**Comment by nelson2005** [comment]
Also confirmed windows 10 pygount v1.4.0, no powershel

**Comment by roskakori** [comment]
Rich 12.6 includes a fix for [#2513](https://github.com/Textualize/rich/pull/2513) which might be related to this. In summary, running python using `pythonw` results in `stdout` being `None` and nothing to be rendered.

**Comment by roskakori** [comment]
During the holidays I decided to install Python on my Windows 10 box that otherwise is exclusively used for gaming and finally track this down.

Turns out the bug shows as soon as the output for `--format=sloccount` is wrapped between:
```python
            with Progress(transient=True) as progress:
                try:
                    ...
                finally:
                    progress.stop()
```
Even without any `progress.track()`.

I also noticed that using `--format=sloccount` on macOS in the PyCharm terminal results into weird output where tracking information is mixed with SLOC information. This might not be the case on all terminals but I found it discerning enough on this one.

So I decided to disable progress tracking for `--format=sloccount` for good. When writing to the terminal this should not be an issue because each file is written immediately, so there is always something going on. The only annoying case is when redirecting the output to a file. Then one can only stare at no output and wait until it's finished. So there might be a better solution but I'm not going to look into it any further.

The other writers do not have this issue because they collect all information where processing files in the background, and only once all are processed, write their findings. By then, the progress is already at 100% and not updated anymore.
